client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -89,6 +95,11 @@ export function SitesDashboard( { queryParams: { search, status } }: SitesDashbo
 
 	const selectedTabName = tabs.find( ( tab ) => tab.name === status )?.name;
 
+	const { sortedSites } = useSitesTableSorting( filteredSites, {
+		sort: 'last-publish',
```

Comment:
What do you think about calling this `updated-at`?

My personal preference is to name query params, etc. after the data they're referencing, vs. UI. The UI may change, but the underlying data won't. It also makes it more obvious which data the query param is referring to.

Reply:
Thanks, it's a good point, it makes perfect sense. I've changed that to `updated-at`.